addons/stock_maintenance/models/maintenance.py

```diff
@@ -1,5 +1,3 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
```

Comment:
unnecessary change. I forgot what the current policy is about including this, but we'll do a cleaning of all the code if we want to get rid of it